### PR TITLE
fix(context-menu): fix action sheet styles

### LINKED ISSUE #5322: Native Mobile - Fix action sheet styles

### PR DESCRIPTION
**Pull request by nathan-power**
### What this PR does 📖
- Fixes the styling for action sheets in mobile

### Which issue(s) this PR fixes 🔨
- Resolve #5322 

### Special notes for reviewers 🗒️
- To bring up an action sheet long press a list item in the chat list on mobile.
- Styling bug for action sheets opened inside the friends screen will be fixed in a separate PR.

### CODE REVIEW

File: components/ui/ActionSheet/ActionSheet.vue [hunk 1]
```diff
@@ -51,56 +55,53 @@
...
+    return
+  }
+  emit('action', func)
+}
+
+const isVisible = computed(() => props.isVisible)
+
+watch(isVisible, (isVisible) => {
+  $store.commit('ui/setIsMobileNavVisible', !isVisible)
+})
```
Comment (josephmcg) [inline components/ui/ActionSheet/ActionSheet.vue:100]: we saw copilot doing this yesterday, but maybe a function watcher is better if we use props
https://stackoverflow.com/questions/59125857/how-to-watch-props-change-with-vue-composition-api-vue-3

**Review by josephmcg — CHANGES_REQUESTED** [review]
nice, just 1 minor change. we can take a look at the mixin later and remove it

File: components/views/navigation/mobile/nav/Nav.less [hunk 2]
```diff
@@ -1,27 +1,13 @@
 .nav {
+  height: @mobile-nav-height;
   width: 100%;
```
Comment (josephmcg) [inline components/views/navigation/mobile/nav/Nav.less:3]: maybe we dont need this height value since jason adjusted the transitions in a separate PR

Comment (nathan-power) [reply components/views/navigation/mobile/nav/Nav.less:3]: You're right it's not required anymore
I'll remove it now

**Review by josephmcg — CHANGES_REQUESTED** [review]
upon closing context menu on mobile, it feels like there's a second where you can't click anything

**Review by josephmcg — CHANGES_REQUESTED** [review]
this currently renders bottom nav on slide 2 of a chat page once you close the action sheet.

Do we need to hide/show the bottom nav on action sheet render?

**Review by josephmcg — APPROVED** [review]

**Pull request merged**